### ISSUE TITLE
should be using /etc/sysconfig/docker-network for network config

### ISSUE THREAD

**Issue by thoraxe** [openshift/openshift-sdn#52]
There is a file called `/etc/sysconfig/docker-network` that should be used for the network-specific config components.

**Comment by detiber** [comment]
instead of modifying /etc/sysconfig/docker we should be modifying /etc/sysconfig/docker-network instead.
The entries in /etc/sysconfig/docker-network are just appended to the docker options after the entries in /etc/sysconfig/docker and /etc/sysconfig/docker-storage.

- We will need to set `DOCKER_NETWORK_OPTIONS='-b=lbr0 --mtu=1450'` instead of the  OPTIONS var we are setting now (--selinux-enabled is not needed because it is set by default in /etc/sysconfig/docker)
- We will not be able to set the --insecure-registry flag here, since that is overriden with the value from /etc/sysconfig/docker, but we can adjust for that in the ansible playbooks.

**Comment by eparis** [comment]
flannel does the following, first it calls a helper function which generates, at run time, the options you are hard coding into /etc/sysconfig/docker (https://github.com/openshift/openshift-sdn/blob/master/ovs-simple/bin/openshift-sdn-simple-setup-node.sh#L54)
```
ExecStartPost=/usr/libexec/flannel/mk-docker-opts.sh -k DOCKER_NETWORK_OPTIONS -d /run/flannel/docker
```
So that creates /run/flannel/docker with the env variable DOCKER_NETWORK_OPTIONS=   Note that our script is smart enough to prepend previous values in DOCKER_NETWORK_OPTIONS. So it sources /etc/sysconfig/docker-network, rather than overwrite it (https://github.com/coreos/flannel/blob/master/dist/mk-docker-opts.sh) ....


---

Next flannel has a config file:
```
# cat /usr/lib/systemd/system/docker.service.d/flannel.conf
[Service]
EnvironmentFile=-/run/flannel/docker
```

Which means that docker will source /run/flannel/docker.

**Comment by rajatchopra** [comment]
Merged #54
